src/routes/boot.ts

```diff
@@ -328,6 +330,48 @@ const getUser = (con: DataSource, userId: string): Promise<User> =>
     ],
   });
 
+const getMarketingCta = async (con: DataSource, userId: string) => {
+  let marketingCta: MarketingCta | null = null;
+  const redisKey = generateStorageKey(
+    StorageTopic.Boot,
+    'marketing_cta',
+    userId,
+  );
+  const rawRedisValue = await getRedisObject(redisKey);
+
+  if (rawRedisValue === 'SLEEPING') {
```

Comment:
Why do we write sleeping?
Should it not just be there or not?

Reply:
Commented earlier about it
https://github.com/dailydotdev/daily-api/pull/1747#discussion_r1522265543